package.json

```
@@ -21,6 +21,7 @@
     "babel-loader": "9.1.0",
     "copy-webpack-plugin": "11.0.0",
     "css-loader": "6.7.2",
+    "dayjs-random": "^1.0.1",
```

```suggestion
    "dayjs-random": "1.0.1",
```
Не должно быть модификаторов